### DISCUSSION TITLE
Backup Configurations

### DISCUSSION THREAD

**Discussion by WilliamKWusik** [mriscoc/Ender3V2S1#903] [Ideas]
Is there a way to backup the configurations to the SD Card or some other way to make a copy of it? As when I try to put my own compiled firmware it erases the configuration somehow. Still trying to figure out why.

Also, off topic, but when I compiled the firmware for the same settings I had the current one running, the Tool Bar didn't show items anymore. But would work as usual. So I could browser to a location and click to do what should be there, but now is invisible. ;-) 

Cheers and thanks again for such a superb piece of work. :-) <3 ❤️

**Comment by WilliamKWusik** [comment]
I'm currently using: Ender3V2-422-BLTUBL-MPC-20230312

When I compiled, I used the following configuration file: Ender3V2-422-BLTUBL-MPC 
(from the special configurations folder)

**Comment by TheWebMachine** [comment]
You preserve and restore settings via a config.gco file that is loaded up just like a print job after you've flashed new firmware. A firmware flash always wipes settings and the config file is *the* way to carry your settings over. 

https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files

Edit: I just realized you may mean compile configs...sorry if I misunderstood the question.

**Reply by WilliamKWusik** [reply]
Thank you so much, sorry that I overlooked that WiKi page. I did read a ton of stuff, but I'm partially dyslexic, so I tend to overlook things. So, my question was about 2 things, one you replied. The other is why the tool bar menu didn't show when I compiled my own firmware. It works as it was there, invisible, but doesn't show the text. ;-) Thanks again.

**Reply by mriscoc** [reply]
Try with the configuration files unchanged, then customize them change by change.
